lib/include/elements/element/status_bar.hpp

```diff
@@ -117,6 +121,7 @@ namespace cycfi::elements
       void                    animate(view& view_);
 
       double                  _start_pos;
+      double                  _animation_width;
```

Comment:
I've been thinking about `double` vs `float` since my last refactor. Perhaps we need to refactor these indicators to use `float` instead? `float` ought to be enough to save space, esp. now that we have more.

Reply:
It will only be a value between 0 and 1, so a float should be sufficient.